cmd/terramate/cli/cli_run_test.go

```diff
@@ -483,3 +484,15 @@ func TestRunOrderAllChangedStacksExecuted(t *testing.T) {
 		mainTfFileName,
 	), runExpected{Stdout: wantRun})
 }
+
+func TestRunLogsUserCommand(t *testing.T) {
+	s := sandbox.New(t)
+
+	stack := s.CreateStack("stack")
+	testfile := stack.CreateFile("test", "")
+
+	cli := newCLIWithLogLevel(t, s.RootDir(), "info")
+	assertRunResult(t, cli.run("run", "cat", testfile.Path()), runExpected{
+		StderrRegex: `cmd="cat /`,
+	})
+}
```

Comment:
not sure we need a test for this ;)

Reply:
if it's not tested eventually someone not aware of this feat request will change this ...